doc/user/content/guides/cdc-postgres.md

```diff
@@ -0,0 +1,260 @@
+
+* `oid`: A unique identifier for the tables included in the publication;
+
+* `row_data`: A text-encoded, variable length `list`. The number of text elements in a list is always equal to the number of columns in the upstream table.
+
+The next step is to break down this source into views that reproduce the publication’s original tables and can be used as a base for your materialized views.
+
+#### Create replication views
+
+Once you've created the Postgres source, you can create views that filter the replication stream based on the `oid` identifier and convert the text elements in `row_data` to the original data types:
+
+_Create views for specific tables included in the Postgres publication:_
+
+```sql
+CREATE VIEWS FROM SOURCE mz_source (view1, view2);
+```
+
+_Create views for all tables:_
+
+```sql
+CREATE VIEWS FROM SOURCE mz_source;
+```
+
+Under the hood, Materialize parses this statement into view definitions for each table (so you don't have to!).
+
+### Create a materialized view
+
+Any materialized view defined on top of this source will be incrementally updated as new change events stream in, as a result of `INSERT`, `UPDATE` and `DELETE` operations in the original Postgres database.
+
+```sql
+CREATE MATERIALIZED VIEW cnt_view1 AS
+SELECT field1,
+       COUNT(*) AS cnt
+FROM view1
+GROUP BY field1;
+```
+
+## Kafka + Debezium
+
+If Kafka is part of your stack, you can use [Debezium](https://debezium.io/) and the [Kafka source](/sql/create-source/avro-kafka/#debezium-envelope-details) to propagate CDC data from Postgres to Materialize. Debezium captures row-level changes resulting from `INSERT`, `UPDATE` and `DELETE` operations in the upstream database and publishes them as events to Kafka using Kafka Connect-compatible connectors.
+
+### Setup
+
+**Minimum requirements:** Postgres 10+
+
+Before deploying a Debezium connector, you need to ensure that the upstream database is configured to support [logical replication](https://www.postgresql.org/docs/10/logical-replication.html). As a _superuser_:
+
+1. Check the [`wal_level` configuration](https://www.postgresql.org/docs/current/wal-configuration.html) setting:
+
+    ```sql
+    SHOW wal_level;
+    ```
+
+    The default value is `replica`. For CDC, you'll need to set it to `logical` in the database configuration file (`postgresql.conf`). Keep in mind that changing the `wal_level` requires a restart of the Postgres instance and can affect database performance.
+
+    **Note:** Additional steps may be required if you're using Postgres on [Amazon RDS](https://debezium.io/documentation/reference/1.6/connectors/postgresql.html#postgresql-on-amazon-rds).
+
+1. Grant enough privileges to ensure Debezium can operate in the database. This will depend on how much control you want to give to the replication user, so we recommend following the [Debezium documentation](https://debezium.io/documentation/reference/connectors/postgresql.html#postgresql-replication-user-privileges).
```

Comment:
The antecdent of "This" in the second sentence is unclear, so maybe "The privileges included can vary depending on how much control you want to give the replication user. We recommend following the Debezium guidelines."